### DISCUSSION TITLE
Ohm's Law C example

### DISCUSSION THREAD

**Discussion by andy5995** [KaruroChori/vs-fltk#22] [Ideas]
@KaruroChori Would you be interested in a PR with an "Ohm's Law" C example after #20 is fixed?

**Comment by KaruroChori** [comment]
That would be great!
Still, the integration of fltk widgets is lagging a bit behind, as right now I am giving priority to "core features", documentation and automatic pipelines.
I think there should be almost enough functionality to do what you are suggesting, but the "good" widgets to set up a layout like grids or flex are only half implemented.  
And more importantly, getting information back from widgets via computed values (plus fully working getters/setters) was the objective of 0.1.3 so it is not fully done yet.  

I would suggest to revisit this idea next week when it is likely going to be much more feasible :D.
